src/CoreMedia/CMSync.cs

```diff
@@ -147,7 +171,40 @@ public CMTimebase (CMTimebase masterTimebase)
 
 			CFObject.CFRetain (Handle);
 		}
+		
+		[Watch (6,0), TV (9,0), Mac (10,8), iOS (6,0)]
+		[DllImport(Constants.CoreMediaLibrary)]
+		static extern CMTimebaseError CMTimebaseCreateWithSourceClock (/* [NullAllowed] CFAllocatorRef */ IntPtr allocator, /* CMClock */ IntPtr sourceClock, /* CMTimebase */ out IntPtr  timebaseOut);
+
+		[Watch (6,0), TV (9,0), Mac (10,8), iOS (6,0)]
+		public CMTimebase (CFAllocator? allocator, CMClock sourceClock)
+		{
+			if (sourceClock == null)
+				throw new ArgumentNullException (nameof(sourceClock));
+
+			var error = CMTimebaseCreateWithSourceClock (allocator.GetHandle (), sourceClock.Handle, out handle);
+			if (error != CMTimebaseError.None)
+				throw new ArgumentException (error.ToString ());
+
+			CFObject.CFRetain (Handle);
```

Comment:
This isn't necessary, the `handle` parameter is marked as `CM_RETURNS_RETAINED_PARAMETER` in the header.